robolectric/src/main/java/org/robolectric/internal/bytecode/InstrumentationConfiguration.java

```diff
@@ -198,6 +198,7 @@ public boolean shouldAcquire(String name) {
             || name.startsWith("org.xml.")
             || name.startsWith("org.junit")
             || name.startsWith("org.hamcrest")
+            || name.startsWith("org.scalatest")
```

Comment:
What would you say if we add check for packages agains `classesToNotAcquire` in `InstrumentationConfiguration.shouldAcquire` method. This check could look like that:
```
      if (classesToNotAquire.contains(pkgName)) return false;
      while (pkgName.contains(".")) {
          pkgName = pkgName.substring(0, pkgName.lastIndexOf('.'));
          if (classesToNotAquire.contains(pkgName)) return false;
      }
```